relational_operators/TextScanOperator.cpp

```diff
@@ -663,11 +664,16 @@ void TextSplitWorkOrder::sendBlobInfoToOperator(StorageManager *storage_manager,
   std::free(message_proto_bytes);
 
   // Send new work order available message to Foreman.
-  QueryExecutionUtil::SendTMBMessage(
-      bus_,
-      ClientIDMap::Instance()->getValue(),
-      foreman_client_id_,
-      std::move(tagged_message));
+  const tmb::MessageBus::SendStatus send_status =
+      QueryExecutionUtil::SendTMBMessage(
+          bus_,
+          ClientIDMap::Instance()->getValue(),
+          foreman_client_id_,
+          std::move(tagged_message));
+  CHECK(send_status != tmb::MessageBus::SendStatus::kOK) << "Message could not "
```

Comment:
Also, is ``CHECK_EQ`` better over ``CHECK``. I think it's a matter of preference.